ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/PullQueryRow.java

```diff
@@ -27,6 +29,7 @@
   private final LogicalSchema schema;
   private final Optional<KsqlNode> sourceNode;
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP2")
   public PullQueryRow(
       final List<?> row,
```

Comment:
Not sure why `row` must be mutable... But if I change it, some tests break...

Reply:
Thanks for verifying. So just adding a suppression for now seems the right way to go. We can still refactor with a follow up PR to avoid the mutable object and suppression if we think it's worth it.